02-express-tutorial/09-middleware-use.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const app = express()
+const logger = require('./logger')
+const authorize = require('./authorize')
+//  req => middleware => res
+app.use([authorize, logger])
```

Comment:
fwiw, just make sure you understand that the order of middleware is important - the example logs each request before it authorizes, while this order authorizes first so nothing is ever logged if the request is unauthorized.